.github/actions/build/action.yaml

```diff
@@ -48,7 +44,7 @@ runs:
 
     - name: Build
       if: ${{ startsWith(inputs.os, 'ubuntu') }}
-      uses: ./build-checkout/.github/actions/build-ubuntu
+      uses: ${{ inputs.src-dir }}/.github/actions/build-fedora
```

Comment:
is this a copy/paste error in changing ubuntu to fedora?

Reply:
Derp. Good catch. Fixed.